### PR TITLE
[All] Formatting.

### PR DESCRIPTION
**Pull request by willlarche**
PLEASE CHECK YOUR COMPONENTS!

### CODE REVIEW

**Review by samnm — APPROVED** [review]

File: components/Dialogs/src/MDCAlertController.m [hunk 1]
```diff
@@ -398,8 +398,7 @@ - (void)viewWillLayoutSubviews {
         UIUserInterfaceLayoutDirectionRightToLeft) {
       for (UIButton *button in self.actionButtons) {
         CGRect buttonRect = button.frame;
-        CGRect flippedRect = MDCRectFlippedForRTL(buttonRect,
-                                                  CGRectGetWidth(self.view.bounds),
+        CGRect flippedRect = MDCRectFlippedForRTL(buttonRect, CGRectGetWidth(self.view.bounds),
```
Comment (ianegordon) [inline components/Dialogs/src/MDCAlertController.m:401]: I believe once you break up parameters on a line the recommended practice is to have them each on their own individual line.

File: components/FeatureHighlight/src/private/MDCFeatureHighlightLayer.m [hunk 2]
```diff
@@ -24,8 +24,8 @@ - (void)setCenter:(CGPoint)center {
   _center = center;
 
   CGRect circleRect = CGRectMake(center.x - _radius, center.y - _radius, _radius * 2, _radius * 2);
-  self.path = (__bridge CGPathRef _Nullable)
-      (CFBridgingRelease(CGPathCreateWithEllipseInRect(circleRect, NULL)));
+  self.path = (__bridge CGPathRef _Nullable)(
```
Comment (ianegordon) [inline components/FeatureHighlight/src/private/MDCFeatureHighlightLayer.m:27]: I would argue that the original was more readable.

File: components/FeatureHighlight/src/private/MaterialFeatureHighlightStrings.h [hunk 3]
```diff
@@ -1,5 +1,6 @@
 // GENERATED CODE: DO NOT EDIT BY HAND
-// GENERATED FROM: components/FeatureHighlight/src/MaterialFeatureHighlight.bundle/Resources/en.lproj/MaterialFeatureHighlight.strings
```
Comment (ianegordon) [inline components/FeatureHighlight/src/private/MaterialFeatureHighlightStrings.h:2]: This code was generated, we likely want to omit generated code from the formatter.

**Review by ianegordon — COMMENTED** [review]
If our formatter doesn't play nicely with the Google Style Guide or breaks comments in auto-generated code, we may get into trouble. :/

**Comment by willlarche** [comment]
I guess we have to make a hard choice on this formatter.

**Pull request merged**